lib/puppet/provider/splunk_output/ini_setting.rb

```diff
@@ -7,9 +7,9 @@
   def self.file_path
     case Facter.value(:osfamily)
     when 'windows'
-      'C:\Program Files\Splunk\etc\system\local\outputs.conf'
+      'C:\Program Files\Splunk\etc\apps\search\local\outputs.conf'
```

Comment:
http://docs.splunk.com/Documentation/Splunk/6.2.3/Forwarding/Configureforwarderswithoutputs.confd#Custom_versions indicates that both of these paths are correct under some conditions and that the difference in the path is determined "on the type of forwarder and other factors."

Additionally, it says that "it is recommended that you work with just a single copy of the file, which you place in $SPLUNK_HOME/etc/system/local/."

As I don't know how splunk works and don't use it, it would be helpful if you could read that page and let me know what the best decision would be.